kithara/__init__.py

```diff
@@ -112,7 +112,7 @@ def _install_jetstream():
             print(f"Failed to install JetStream: {e}")
 
     jetsteam_dir = Path(
-        os.path.join(os.path.dirname(Path(__file__)), "model/maxtext/JetSteam/jetstream")
+        os.path.join(os.path.dirname(Path(__file__)), "model/maxtext/JetStream/jetstream")
```

Comment:
I'll suggest to use:

```python
jetsteam_dir = Path(__file__).parent / "model/maxtext/JetStream/jetstream"
```

NOTE: I believe you may also want to rename `jetsteam_dir` to `jetstream_dir`?